tests/functional_test.go

```diff
@@ -997,6 +998,13 @@ func GRPCListObjectsTest(t *testing.T, client openfgav1.OpenFGAServiceClient) {
 }
 
 func GRPCListUsersValidationTest(t *testing.T, client openfgav1.OpenFGAServiceClient) {
+	_101tuples := make([]*openfgav1.TupleKey, 101)
```

Comment:
Why is this here? Doesn't seem relevant to this PR as contextual tuples are being handled elsewhere.

Reply:
Then coordinate with @rhamzeh since this won't be the only PR affected.

Reply:
@elbuo8 opened #2041 I'll rebase after it's merged